client/src/components/Map/MapCards.jsx

```diff
@@ -1,15 +1,35 @@
-import React from "react"
+import React, { Component } from "react";
+
 import SingleMapCard from "./SingleMapCard";
 
-const MapCards = (props) => {
+/*global google*/
+
+class MapCards extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
are you expecting to add state to this later or did you forget to remove this and use a functional component instead?